config/activity_insight.yml

```diff
@@ -12,6 +12,6 @@ production:
     :key: "<%= ENV.fetch('FAMS_METADATA_DB_KEY') %>"
   s3_bucket:
     api_key: "<%= ENV.fetch('FAMS_S3_BUCKET_API_KEY') %>"
-  lp_sftp:
+  fams_lp_sftp:
```

Comment:
this can stay lp_sftp -- the code is expecting it to be this. everything else looks fine